draft-ietf-quic-recovery.md

```diff
@@ -1047,6 +1045,19 @@ paces the sending of any packets in excess of the initial congestion window.
 A sender MAY implement alternate mechanisms to update its congestion window
 after idle periods, such as those proposed for TCP in {{?RFC7661}}.
 
+## Application Limited Sending
+
+The congestion window should not be increased in slow start or congestion
+avoidance when it is not fully utilized.  The congestion window could be
+under-utilized due to insufficient application data to send or flow control
+limits.
+
+When the sender is pacing (see {{pacing}}) packets, the sender may be unable
+to use the full congestion window for a period of time after receiving
+acknowledgements, due to pacing.  In this case, the sender should not consider
+themselves application limited and should allow the congestion window to
+increase.
```

Comment:
Suggested rephrasing:  "A sender that uses pacing (see {{pacing}}) might delay sending of packets and might not fully utilize the congestion window due to this delay.  A sender should not consider itself application limited if it might have utilized the entire congestion window without pacing delay."

"When the sender paces packets (see {{pacing}}), it might not send 
to use the full congestion window for a period of time after receiving
acknowledgements, due to pacing.  In this case, the sender should not consider
themselves application limited and should allow the congestion window to
increase."

Reply:
Accepted with minor tweaks.